### ISSUE TITLE
End line is not new line

### ISSUE THREAD

**Issue by subversivo58** [showdownjs/showdown#405]
I use GFM and this:

    I'm an online text
    And I'm still online

    I'm a new line

Produces this:

    <p>
    "I'm an online text"
    <br>
    "And I'm still online"
    </p>

    <p>
    "I'm a new line"
    </p>

Is that correct? A long text that should be understood with an integer element "inline" ends up being presented as several parts.

Sorry if my reasoning is wrong but your I write in my editor will not follow the same line until the end of a paragraph in case you are writing a long text.

For example here on GitHub would result in this:

    <p>
    "I'm an online text"
    And I'm still online"
    </p>

    <p>
    "I'm a new line"
    </p>

As a result the text will still be "inline". Is this a source question? It could be different or should I try to create an extension?

**EDIT:**

I've found a similar question in [#281](https://github.com/showdownjs/showdown/issues/281) however it refers to the "makeHtml" function which is different since I'm converting "Convert"

**Example:**

![eg1](https://user-images.githubusercontent.com/6475273/27433070-5b9082d2-5729-11e7-8892-0c71131402d6.PNG)

In screenshoot up result this:

    <p>
    "Lorem Ipsum é simplesmente ..."
    </p>

    <p>
    "New line bla bla bla"
    </p>

As shown in the image below:

![eg2](https://user-images.githubusercontent.com/6475273/27433205-09ef9eb2-572a-11e7-89d6-2d85eb99b8b1.PNG)

In **showdown** this result:

    <p>
    "Lorem Ipsum é simplesmente ..."
    <br>
     "século XVI, quando ..."
    <br>
     "modelos de tipos ..."
     <br>
     "more lines (breaks) ..."
    <p>

    <p>
    "New line bla bla bla"
    </p>

In the "html" language the paragraph is an "inline" element in the "markdown" syntax if the content is not preceded by a marker it takes the "inline" style until there is a spacing (which characterizes the line break)

**Comment by tivie** [comment]
## TL;DR

Disable the option [`simpleLineBreaks`](https://github.com/showdownjs/showdown/wiki/Showdown-options#simplelinebreaks)

```js
converter.setOption('simpleLineBreaks', false);
```

## Explanation:

Github uses simple line breaks in most places, like comments:

```
This is a wrapped
text line
```

This is a wrapped
text line

By default, showdown does not beahve like this. However, when you use GFM flavor, `simpleLineBreaks` gets activated. So you need to deactivate that option again.

**Comment by subversivo58** [comment]
I got lost ... I went through the "wiki" but found reference that it was necessary to deactivate again if I used "GFM"

Alright thank you, fine works!
